cmd/nerdctl/image_push_linux_test.go

```diff
@@ -174,3 +176,69 @@ func TestPushNonDistributableArtifacts(t *testing.T) {
 	}
 	assert.Equal(t, resp.StatusCode, http.StatusOK, "non-distributable blob should be available")
 }
+
+func TestPushSoci(t *testing.T) {
```

Comment:
test LGTM. maybe could deduplicate some of https://github.com/containerd/nerdctl/blob/main/cmd/nerdctl/container_run_soci_linux_test.go